examples/ads.amp.html

```diff
@@ -77,13 +77,6 @@
            run a local server.
         2) keep the list in alphabetic order
     -->
-    <a href="#a9">A9</a> |
```

Comment:
Should probably not delete these in the official ads demo file.  Better to make a local-only demo file for yourself.  (With a bit of git config tweaking, you can make local files that git will ignore.)